### PR TITLE
expanded example and some adjustments from connect

### PR DESCRIPTION
**Pull request by aronatkins**
1. improve the README.
2. Improve the example site to have hybrid sidebar navigation.
3. remove `webside.sidebar` navigation from the extension, since that prevented hybrid sidebar navigation. other than `collapse-level` and `search`, this only provided default values.
4. add `format: posit-docs-html` to the extension, following the example at https://quarto.org/docs/extensions/project-types.html
5. add "small" styling for the product version in the navbar.
6. enable search copy-button and search show-item-content.

Projects will need to explicitly configure `sidebar.collapse-level` and `sidebar.search` if they need it.

### CODE REVIEW

File: _extensions/posit-docs/theme.scss [hunk 1]
```diff
@@ -80,6 +80,13 @@ $list-group-color: $primary !default;
     font-weight: 500;
 }
 
+/* Style for the version included in the website title */
+.navbar-title small {
+    font-size: 0.6em;
+    display: block;
+    padding-left: 1em;
+}
+
```
Comment (AshleyHenry15) [inline _extensions/posit-docs/theme.scss:89]: This doesn't meet accessibility standards. 
If you remove this and add it to an issue, I can play around with this when I am working through some of the dark theme changes.

File: _extensions/posit-docs/_extension.yml [hunk 2]
```diff
@@ -21,11 +20,10 @@ contributes:
...
              href: "https://support.posit.co/hc/en-us/"
-      sidebar:
-        style: "floating"
-        collapse-level: 1
-        search: true
-        pinned: false
+      search:
+        copy-button: true
+        show-item-context: true
+    format: posit-docs-html
```
Comment (AshleyHenry15) [inline _extensions/posit-docs/_extension.yml:26]: Why was this added? We ran ito issues with having format and html included before and had to remove it.

**Review by AshleyHenry15 — CHANGES_REQUESTED** [review]

File: _extensions/posit-docs/theme.scss (hunk 1 above)
Comment (aronatkins) [reply _extensions/posit-docs/theme.scss]: Adjusted.

File: _extensions/posit-docs/_extension.yml (hunk 2 above)
Comment (aronatkins) [reply _extensions/posit-docs/_extension.yml:26]: Sets a default format for users of this custom project type. Problems can happen when folks try to provide overrides to both `format: html` and `format: posit-docs-html`; they can use only one.

**Review by AshleyHenry15 — APPROVED** [review]
